src/main/groovy/io/seqera/wave/service/metric/impl/MetricsServiceImpl.groovy

```diff
@@ -52,13 +52,13 @@ class MetricsServiceImpl implements MetricsService {
     @Override
     GetOrgCountResponse getAllOrgCount(String metric){
         final response = new GetOrgCountResponse(metric, 0, [:])
-        final orgCounts = metricsCounterStore.getAllMatchingEntries("$metric/$MetricConstants.PREFIX_ORG/*")
+        final orgCounts = metricsCounterStore.getAllMatchingEntries("$metric/$MetricsConstants.PREFIX_ORG/*")
         for(def entry : orgCounts) {
             // orgCounts also contains the records with org and date, so here it filter out the records with date
-            if(!entry.key.contains("/$MetricConstants.PREFIX_DAY/")) {
+            if(!entry.key.contains("/$MetricsConstants.PREFIX_DAY/")) {
                 response.count += entry.value
                 //split is used to extract the org name from the key like "metrics/o/seqera.io" => seqera.io
-                response.orgs.put(entry.key.split("/$MetricConstants.PREFIX_ORG/").last(), entry.value)
+                response.orgs.put(entry.key.split("/$MetricsConstants.PREFIX_ORG/").last(), entry.value)
```

Comment:
Using a `import static MetricsConstants.*` would improve the readability of this